web/app.js

```diff
@@ -205,10 +205,10 @@ const PDFViewerApplication = {
       if (mode) {
         document.documentElement.classList.add(mode);
       }
-    } else {
+    } else if (AppOptions.get("enableAltText")) {
```

Comment:
This means that `this.mlManager` can now be unset in Firefox in which case the following code will throw:
https://github.com/mozilla/pdf.js/blob/5184a38e3de4e2e4065df30019dd48501ef59962/web/app.js#L379-L386
Hence that needs to be replaced with:
```javascript
      eventBus = AppOptions.eventBus = new FirefoxEventBus(
        AppOptions.get("allowedGlobalEvents"),
        externalServices,
        AppOptions.get("isInAutomation")
      );

      if (this.mlManager) {
        this.mlManager.eventBus = eventBus;
      }
```

Reply:
Thank you for noticing this.